transportation.sk

```diff
@@ -122,14 +122,21 @@ the wild update:
 
 	# boat with chest
 	spruce (chest boat¦s|boat¦s with chest) = minecraft:spruce_chest_boat[relatedEntity=spruce chest boat]
+	any spruce boat¦s = spruce boat, spruce chest boat
 	birch (chest boat¦s|boat¦s with chest) = minecraft:birch_chest_boat[relatedEntity=birch chest boat]
+	any birch boat¦s = birch boat, birch chest boat
 	jungle (chest boat¦s|boat¦s with chest) = minecraft:jungle_chest_boat[relatedEntity=jungle chest boat]
+	any jungle boat¦s = jungle boat, jungle chest boat
 	acacia (chest boat¦s|boat¦s with chest) = minecraft:acacia_chest_boat[relatedEntity=acacia chest boat]
+	any acacia boat¦s = acacia boat, acacia chest boat
 	oak (chest boat¦s|boat¦s with chest) = minecraft:oak_chest_boat[relatedEntity=oak chest boat]
+	any oak boat¦s = oak boat, oak chest boat
 	dark oak (chest boat¦s|boat¦s with chest) = minecraft:dark_oak_chest_boat[relatedEntity=dark oak chest boat]
-	mangrove (chest boat¦s|boat¦s with chest) = minecraft:mangrove_chest_boat[relatedEntity=mangrove chest boat]
-	[any] (chest boat¦s|boat¦s with chest) = spruce chest boat, birch chest boat, jungle chest boat, acacia chest boat, dark oak chest boat, mangrove chest boat
+	any dark oak boat¦s = dark oak boat, dark oak chest boat
 
-	# mangrove
 	mangrove boat¦s = minecraft:mangrove_boat[relatedEntity=mangrove boat]
-	[any] boat¦s = any boat, mangrove boat
\ No newline at end of file
+	mangrove (chest boat¦s|boat¦s with chest) = minecraft:mangrove_chest_boat[relatedEntity=mangrove chest boat]
+	any mangrove boat¦s = mangrove boat, mangrove chest boat
+
+	[any] {wood type} (chest[ed] boat¦s|boat¦s with [a] chest) = spruce chest boat, birch chest boat, jungle chest boat, acacia chest boat, dark oak chest boat, mangrove chest boat

```

Comment:
```suggestion
	[any] (chest[ed] boat¦s|boat¦s with [a] chest) = spruce chest boat, birch chest boat, jungle chest boat, acacia chest boat, dark oak chest boat, mangrove chest boat
```
We don't need a wood type specifier here since this is *any* type of wood boat